src/app/Hooks.ts

```diff
@@ -446,10 +446,24 @@ const useSearchFilters = () => {
         return searchParams.getAll(name).includes(value);
     };
 
+    /**
+     * Function to reset and remove all search filters
+     */
+    const ResetSearchFilters = () => {
```

Comment:
I think it would be more efficient if you returned a new empty list of search params instead of deleting every search param in a loop

Reply:
Would be nice, but unfortunately there is no function for that

Reply:
Also the reason why all the other functions in this hook do the deleting first, upon appending the search filters individually